websites/docs-smoke-test/gatsby-config.js

```diff
@@ -14,6 +14,7 @@ module.exports = {
       options: {
         websiteKey: 'docs-smoke-test',
         excludeFromSearchIndex: isProd && !shouldEnableSearch,
+        additionalPrismLanguages: ['scala', 'csharp'],
```

Comment:
Now we get proper syntax highlighting 💅

<img width="830" alt="image" src="https://user-images.githubusercontent.com/1110551/77798159-a4b30600-7072-11ea-941c-313dd3a9ca21.png">